jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/SQSManager.java

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ * Copyright 2023 Paladin Cloud, Inc. or its affiliates. All Rights Reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.  You may obtain a copy
+ * of the License at
+ * <p>
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ ******************************************************************************/
+package com.tmobile.cso.pacman.datashipper.entity;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicSessionCredentials;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+import com.amazonaws.services.sqs.model.SendMessageRequest;
+import com.amazonaws.services.sqs.model.SendMessageResult;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.tmobile.cso.pacman.datashipper.config.CredentialProvider;
+import com.tmobile.cso.pacman.datashipper.dto.JobSchedulerSQSMessageBody;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SQSManager {
```

Comment:
do we have similar manager somewhere else? if functionality is limited by just sending SQS messages - I think it should be move to common and reused.

Reply:
Yes, But its in admin API. since they have different common packages, we cannot reuse them

Reply:
got it.